### ISSUE TITLE
"Reload app from server" menu item

### ISSUE THREAD

**Issue by szhu** [rsms/fb-mac-messenger#110]
- This should be called "Reload App from Server" or "Reload App" or simply "Reload". The lowercase looks weird and doesn't reflect Apple's guidelines.
- Even though it's reloading the whole app, I feel like it belong more in the "View" menu than in the app menu.

**Comment by rsms** [comment]
1. I prefer International English to US English. Apple HIG does not say you must use US English. So wontfix
2. Na, it's really the app. Messenger.com represents about 99.99% of the code in this app.

**Comment by szhu** [comment]
> I prefer International English to US English. Apple HIG does not say you must use US English. So wontfix

I'm not too sure what you mean by this. I think you are saying that you are following international English HIG guidelines. If that is the case, I don't think that is correct.

I am pointing out that the menu item (1) is in lowercase and (2) might be too wordy. (I don't think 2 is true anymore.) Neither of these matters depends on which variation of English you use. All menu items on Mac are typically word-capitalized:  
![screen shot 2015-04-27 at 11 42 22 pm](https://cloud.githubusercontent.com/assets/1570168/7364073/158f97d6-ed37-11e4-8cea-05c99e1c51d4.png)

> > Even though it's reloading the whole app, I feel like it belong more in the "View" menu than in the app menu.
> 
> Na, it's really the app. Messenger.com represents about 99.99% of the code in this app.

I know it technically is, but to the end user, they are really just reloading the current view. Therefore I believe it belongs in the same place as where Reload is found in most web browsers – in the View menu.

Also, imagine if a future version of this app supports having multiple windows, and you can individually reload windows. In that hypothetical situation, it would be easier to ~~image~~ imagine that Reload belongs in the View menu.

**Comment by iamgp** [comment]
I agree with @szhu  on both accounts. I hadn't seen that menu item at all till I had to search for it. Regardless of where it is, it should be capitalised.
